src/test/scala-2.12/org/apache/james/gatling/simulation/JmapReadOnlySimulation.scala

```diff
@@ -0,0 +1,30 @@
+package org.apache.james.gatling.simulation
+
+import org.apache.james.gatling.control._
+import org.apache.james.gatling.control.AuthenticatedUserFeeder._
+import org.apache.james.gatling.jmap.scenari.JmapReadOnlyScenario
+
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+
+class JmapReadOnlySimulation extends Simulation {
+
+  private def recordValueToString(recordValue: Any): String = recordValue match {
+    case s: String => s
+    case a: Any => println("Warning: calling toString on a feeder value"); a.toString
+  }
+
+  val authenticatedUsers: Iterator[AuthenticatedUser] = csv("usersAuthenticated.csv").readRecords
```

Comment:
Where is that csv file?

Reply:
As it may contain some real users we should not commit it here o avoid some information disclosure, I guess. Maybe we could provide an exemple?

Reply:
Was not sure if it would be real data or not :)

Reply:
I have just set the filename, no leak here.